packages/streamline/src/components/cards/card-carousel/card-carousel.tsx

```diff
@@ -0,0 +1,121 @@
+import { LinearGradient } from 'expo-linear-gradient';
+import React from 'react';
+import { Image, StyleSheet } from 'react-native';
+
+import { usePress } from '../../../hooks/use-press.hook';
+import { Box } from '../../../primitives/box/box';
+import { Card } from '../../../primitives/card/card';
+import { Icon } from '../../../primitives/icon/icon';
+import { Text } from '../../../primitives/text/text';
+import ButtonIcon from '../../buttons/button-icon/button-icon';
+import Tag from '../../tag/tag';
+import { CardCarouselSkeleton } from './card-carousel-skeleton';
+import {
+  LARGE_CARD_HEIGHT,
+  LINEAR_END_X,
+  LINEAR_START_X,
+  SMALL_CARD_SIZE,
+} from './card-carousel.constants';
+import { CardCarouselProps } from './card-carousel.types';
+
+export const CardCarousel = (props: CardCarouselProps) => {
+  const {
+    size,
+    title,
+    description,
+    onPress,
+    onPressIn,
+    onPressOut,
+    isSkeleton,
+    accessibilityLabel,
+    isLoading,
+    testID,
+    media,
+    onClose,
+  } = props;
+
+  const isSmall = size === 'small';
+  const cardHeight = isSmall ? SMALL_CARD_SIZE : LARGE_CARD_HEIGHT;
+  const cardWidth = isSmall ? SMALL_CARD_SIZE : '100%';
+
+  const [handlePress, isResolving] = usePress({ onPress: onPress });
+
+  if (isSkeleton) return <CardCarouselSkeleton size={size} testID={testID} />;
+
+  return (
+    <Card
+      height={cardHeight}
+      width={cardWidth}
+      flexDirection="column"
+      justifyContent="space-between"
+      overflow="hidden"
+      onPress={handlePress}
+      onPressIn={onPressIn}
+      onPressOut={onPressOut}
+      testID={testID}
+      accessibilityLabel={accessibilityLabel}
+    >
+      {media ? (
+        <Image source={media} style={StyleSheet.absoluteFillObject} />
+      ) : null}
+
+      <LinearGradient
+        colors={['rgba(0, 0, 0, 0.00)', 'rgba(0, 0, 0, 0.80)']}
```

Comment:
You could also extract the colors as constants

Reply:
Changed as const

Reply:
@hcourthias For this comment, do you prefer change it with only one const, by exemple LINEAR_BACKGROUND.
Or two different const ?

Reply:
I do not have preference

Reply:
I do not have preference

Reply:
Change with a LINEAR_BACKGROUND const